src/components/Input/_input.scss

```diff
@@ -23,6 +23,10 @@ $lg: #{$px}-input--lg;
     &--hidden {
       @include hidden;
     }
+
+    @media (min-width: $breakpoint-md) {
+      font-size: 1rem;
```

Comment:
it seems to me that the input label (and string inside it) should be rendered with a `<Text variant={TextVariants.string2}`?
https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=7584-17569&m=dev

![image](https://github.com/user-attachments/assets/4d894c40-2fe0-4031-bada-2642764172a7)

Reply:
will that handle the breakpoints?

Reply:
or I guess I just use the fresnel thing